components/ProductDetail/ProductDetail.js

```diff
@@ -168,11 +155,12 @@ class ProductDetail extends Component {
           optionId: selectedOption ? selectedOption._id : null,
           action: PRODUCT_ADDED
         });
+        */
 
         // The mini cart popper will open automatically after adding an item to the cart,
         // therefore, a CART_VIEWED event is published.
         // debugger // eslint-disable-line
-        this.trackCartItems({ cartItems: items, cartId: cart._id, action: CART_VIEWED }); // eslint-disable-line camelcase
+        // this.trackCartItems({ cartItems: items, cartId: cart._id, action: CART_VIEWED }); // eslint-disable-line camelcase
```

Comment:
Ah, yes, I wanted to discuss this, but didn't investigate options yet, I agree these should definitely be adressed prior merging.
It became clear that the current approach with the tracking decorators wouldn't really be suitable for further development, and also was not really comfortable to work with.

During removal of the custom babel configuration I had not decided yet what to do with them, but did not want to remove all the references to tracking functionality upfront, in case there might be a simple way to adapt them.

Im open for suggestions here - I could just remove them from now, as we could also lookup references from the existing branches - Also, lot's of them seem obsolete and could probably better be abstracted in the hook for the respective action instead.

In the longer run, I would need to research how to use react-tracking with hooks now, and maybe also evaluate if it still is a good choice.
Would tracking be a priority for an initial merge?

Reply:
Let's remove the current tracking implementation and create an issue to re-implement using a modern hooks based approach.

Reply:
Removed in https://github.com/reactioncommerce/example-storefront/pull/667/commits/f99f9adf797a20148632641ef9413b915e711c56